README.md

```diff
@@ -320,6 +323,34 @@ tube_evaluator = TubeEvaluator(annot_filepath, preds_filepath)
+| --img           | Takes in a directory where the corresponding images are located                                                                                       | python cli.py --img toyexample/images                        |         | Used in some ground truth annotation formats. At the current moment, there is visualization functionality for the CLI  |
+| --gtformat      | Takes in keyword that represents the format that the groundtruth will be in (example: COCO GT JSON format)                                            | python cli.py --gtformat coco                                |         | Available formats: `coco`, `voc`, `imagenet`, `labelme`, `openimg`, `yolo`, `absolute`, `cvat`, `tube`                 |
+| --detformat     | Takes in keyword that represents the format that the detection will be in (either text files or COCO Detection JSON format)                           | python cli.py --detformat coco                               |         | Available inputs: `coco`, `xyrb` (x,y,x2,y2), or `xywh` (x,y,width,height)                                             |
+| --metrics       | Takes in keyword that represents the metric that will be used                                                                                         | python cli.py --metrics voc2012                              |         |                                                                                                                        |
+| --names, -n     | The path of the file that contains the class names for this detection                                                                                 | python cli.py --names voc_classes.txt                        |         | The name file should just be a plaintext file with each class name delimited by a newline.                             |
+| --threshold, -t | Represents the IoU threshold for a given metric                                                                                                       | python cli.py --metrics voc2007 --threshold 0.75             | 0.5     |                                                                                                                        |
+| --prgraph, -pr  | A flag for creating a Precision x Recall graph for the given metric that it is being used in                                                          | python cli.py --metrics voc2007 -pr                          |         | This flag requires the `savepath` argument to be specified                                                             |
+| --savepath, -sp | An optional parameter used to specify where the visual output will be saved to. Required when `-pr` is specified                                      | python cli.py --savepath toyexample/image_out/               |         |                                                                                                                        |
```

Comment:
what is its default path?
shoundn't it be `--save_path`?

Reply:
I guess the example and notes for that argument are confusing... Right now, `-pr` will error out if `--savepath` is not defined or the path does not exist. `--savepath` does not have a default value at the moment.

A possible default value for it could be the current relative path ("./"). How does that sound?

Reply:
Yes, I think the current relative path ("./") is a good option too. I leave it to you, but we could create a folder in the current relative path to store the files in there, such as ("./results/")

Reply:
Default path "./results/" added for `savepath` in https://github.com/rafaelpadilla/review_object_detection_metrics/pull/57/commits/50d65485d67d6d14e921061c0cf75d8518a648f0